openstack/arc/templates/api-deployment.yaml

```
@@ -42,11 +50,21 @@ spec:
             initialDelaySeconds: 5
             timeoutSeconds: 1
           livenessProbe:
+{{- if .Values.api.tls.enabled }}
+            exec:
+              command:
+                - sh
+                - -c
+                - cmp /tls/tls.crt /tmp/tls.crt.livenesspobe
```

```suggestion
                - cmp --silent /tls/tls.crt /tmp/tls.crt.livenesspobe
```

This will make cmp exit on the first difference and not output anything

---

Hmm, in busybox version only `cmp -s` is available.